packages/integration-tests/src/verify_outputs.ts

```diff
@@ -0,0 +1,40 @@
+import { CloudFormationClient } from '@aws-sdk/client-cloudformation';
+import { AmplifyClient } from '@aws-sdk/client-amplify';
+import * as fs from 'fs/promises';
+import { BackendOutputClientFactory } from '@aws-amplify/deployed-backend-client';
+
+const region = process.env.AWS_REGION;
+const credentials = {
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID ?? '',
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY ?? '',
+  sessionToken: process.env.AWS_SESSION_TOKEN ?? '',
+};
```

Comment:
We should use this: https://github.com/aws-amplify/amplify-backend/blob/main/packages/integration-tests/src/e2e_tooling_client_config.ts

Reply:
This is run from the context of a test project which would have a different path to `e2e_tooling_client_config` from this original file. How about passing credentials and region as inputs to the subprocess that runs this file?

Reply:
can we not import `e2eToolingClientConfig` normally in this file? When the node process executing this script runs, the import should work right?

Reply:
The problem is we copy this file to the test project during creation then when the node process executes this script there is an error on the import because it cannot find the `e2eToolingClientConfig` file.

See https://github.com/aws-amplify/amplify-backend/actions/runs/9620712621/job/26539435462#step:8:1718 where I normally import `e2eToolingClientConfig`.

Reply:
Ah I see. I wonder if it would be better to use a dynamic import of the backend client from npm rather than run this script in a child process? eg
```ts
import { BackendOutputClientFactory as CurrentCodebaseBackendOutputClientFactory } from '@aws-amplify/deployed-backend-client';

const {BackendOutputClientFactory: NpmBackendOutputClientFactory} = await import('path/to/where/the/module/was/installed');

// use / compare results of both clients

```

It seems like this would eliminate the need to copy over this script and run it in a child process. But I might be overly optimistic about how well that dynamic import will work.

Reply:
That's something I can explore since it's worth a shot to try and cut down the complexity of making this assertion

Reply:
The dynamic import seems to have worked, so I removed the parts of code referring to the script